site/docs/components/card/accessibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Keyboard interactions
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Tab">
+
+Move focus through cards (if interactable); or through interactive elements within the cards (if default) in a logical tabbing order.
```

Comment:
```suggestion
Move focus through cards (if interactable); or through interactive elements within the cards (if default) in a logical tab order.
```
as per latest guidance from content

Reply:
Minor update made to the above suggestion